content/tracing/product_specs/metrics_namespace.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Traces metrics namespace
+kind: documentation
+---
+
+The trace metrics namespace is `trace.<name>.<metrics>{<tags>}` where
+
+* `<name>` is coming from the name of the integration ("redis", "pylons", etc.)
```

Comment:
Actually, this is the name of the operation ie `span.name`. For these examples: `redis.command`, `pylons.request`, `rails.request`, `mysql.query`, ...)

Reply:
👌